<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComForm</class>
 <widget class="QWidget" name="ComForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Selecting Port</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="ListPort">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="BTNPortRefresh">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: white;
color: black;
border: 2px solid grey;
border-radius: 3px;
}
QPushButton:pressed{
background-color: grey;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/refreshing.png</normaloff>:/images/refreshing.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BTNOK">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: white;
color: black;
border: 2px solid grey;
border-radius: 3px;
}
QPushButton:pressed{
background-color: grey;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BTNCancle">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: white;
color: black;
border: 2px solid grey;
border-radius: 3px;
}
QPushButton:pressed{
background-color: grey;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="LabelSelected">
         <property name="text">
          <string>Selected :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CBoxBaudrate"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
